components/UpdateEmailAddress/index.tsx

```diff
@@ -0,0 +1,270 @@
+import React, { Dispatch, SetStateAction, useEffect, useRef, useState } from 'react';
```

Comment:
New UpdateEmailAddress component that renders the`Email and Authentication` portion of the user profile page